receiver/k8seventsreceiver/README.md

```diff
@@ -81,6 +81,26 @@ data:
 EOF
 ```
 
+## Extracting attributes from event labels and annotations
+
+The k8seventsreceiver can also set resource attributes from k8s labels and annotations of event.
+This config represents a list of annotations/labels that are extracted from event and added to log.
+Each item is specified as a config of an optional tag_name (representing the tag name to tag the spans with),
+key (representing the key used to extract value) or key_regex (representing the key regex used to extract the value).
+
+A few examples to use this config are as follows:
+
+```yaml
+extract:
+  annotations: # same can be applied for labels
+    - tag_name: a1 # extracts value of annotation from event with key `annotation-one` and inserts it as a tag with key `a1`
+      key: annotation-one
```

Comment:
Can we add a sample produced log as an example here? My concern is mostly about how these will be represented and under which fields of the Log 🤔 .

Reply:
THB, I don't know how as there is no produced log example exist for this receiver to add new attributes to [this set](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/18cebbb2963064446e0d65744bbbd0bc20e968f8/receiver/k8seventsreceiver/k8s_event_to_logdata.go#L71).
Is there a good example to follow?